src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -317,12 +317,51 @@
     </div>
 
     <!-- Connect with us -->
+    {% assign socialLinksObject = fieldClpConnectWithUs.entity.fieldSocialMediaLinks.platformValues | jsonToObj %}
+
     <div class="vads-u-background-color--primary-alt-lightest vads-u-padding-bottom--1">
       <div class="usa-grid usa-grid-full vads-u-padding-y--4 vads-u-padding-x--2 large-screen:vads-u-padding-x--0">
         <p class="vads-u-color--gray-medium vads-u-font-size--sm vads-u-margin-bottom--0">CONNECT WITH US</p>
         <h2 class="vads-u-margin--0">Get updates from Veterans Health Administration</h2>
 
-        <!-- TODO, check out fieldClpConnectWithUs, I don't think it has everything we need but most of it should be there -->
+        <ul class="usa-unstyled-list vads-u-padding-top--4 vads-u-display--flex vads-u-flex-wrap--wrap">
+          <li class="vads-l-col--6 medium-screen:vads-l-col--4 vads-u-margin-bottom--2">
+            <a href="{{ fieldClpConnectWithUs.entity.fieldEmailUpdatesUrl }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
+              <i class="fas fa-envelope vads-u-padding-right--1"></i>
```

Comment:
Can we add `aria-hidden="true"` to icons?

Reply:
Thank you! 🙂 Updated!